src/components/FilterTopPlanter.js

```diff
@@ -3,8 +3,10 @@ import { withStyles } from '@material-ui/core/styles';
 import Grid from '@material-ui/core/Grid';
 import Button from '@material-ui/core/Button';
 import TextField from '@material-ui/core/TextField';
+// import MenuItem from '@material-ui/core/MenuItem';
 import FilterModel from '../models/FilterPlanter';
-import { connect } from 'react-redux';
+// import { ALL_ORGANIZATIONS, ORGANIZATION_NOT_SET } from '../models/Filter';
```

Comment:
Commented out code is an incomplete refactor to offer the same Organization dropdown that is on other filters.  There is a bug when it's implemented: not all the suborganizations seem to get get included when Freetown is selected from the dropdown.